### ISSUE TITLE
Can not use gitmoji behind an HTTP proxy

### ISSUE THREAD

**Issue by teid** [carloscuesta/gitmoji-cli#91]
Hello @carloscuesta!

I ran into an issue when using gitmoji-cli through a HTTP corporate proxy.
It seems to be related to this issue on Axios which has been stuck couple months: https://github.com/axios/axios/pull/959

Would you accept a PR to migrate the `gitmojiApiClient` to another http client ?

Issue: 
When using gitmoji-cli through a HTTP proxy, Axios tries to connect to the proxy as an HTTPS proxy. 

You can reproduce the issue by setting up a squid proxy:
```
docker run --name squid -d --restart=always --publish 3128:3128 sameersbn/squid:3.3.8-23
export HTTPS_PROXY=http://localhost:3128
gitmoji -c
```

**OS**: Linux
**gitmoji-cli -v**: 1.8.8
**Node version**: v9.11.1

**Error Message**: 
> ERROR: Network connection not found - EPROTO

**Comment by aneveux** [comment]
Hello @carloscuesta :)

I hope you're doing fine!

I'm a huge fan of gitmoji and I sadly ran in the same problem as @teid while using gitmoji-cli behind a corporate proxy. I luckily found this issue but I'm so sure about the status of it, do you have any information about that topic?

Thanks a lot, and have an awesome day!

**Comment by cchang62** [comment]
Hi, @carloscuesta  Your gitmoji is such a great work but I also suffer the proxy issue. I think proxy settings are correct. Thank you.

**Comment by xurizaemon** [comment]
It does look like axios/axios#659 has been merged and released in their 0.19.0, from all the snyk mentions in that issue. May be that we now can just update to that release?
